huaweicloud/config/logger.go

```diff
@@ -76,28 +73,23 @@ func (lrt *LogRoundTripper) RoundTrip(request *http.Request) (*http.Response, er
 	for response == nil {
 
 		if retry > lrt.MaxRetries {
-			if lrt.OsDebug {
-				log.Printf("[DEBUG] connection error, retries exhausted. Aborting")
-			}
+			log.Printf("[DEBUG] connection error, retries exhausted. Aborting")
 			err = fmt.Errorf("connection error, retries exhausted. Aborting. Last error was: %s", err)
 			return nil, err
 		}
 
-		if lrt.OsDebug {
-			log.Printf("[DEBUG] connection error, retry number %d: %s", retry, err)
-		}
+		log.Printf("[DEBUG] connection error, retry number %d: %s", retry, err)
+
 		//lintignore:R018
 		time.Sleep(retryTimeout(retry))
 		response, err = lrt.Rt.RoundTrip(request)
 		retry++
 	}
 
-	if lrt.OsDebug {
-		log.Printf("[DEBUG] API Response Code: %d", response.StatusCode)
-		log.Printf("[DEBUG] API Response Headers:\n%s", FormatHeaders(response.Header, "\n"))
+	log.Printf("[DEBUG] API Response Code: %d", response.StatusCode)
```

Comment:
shall we print the API request and response only when level is **TRACE**?

Reply:
change to TRACE will affect original users, suggest to keep DEBUG here.

Reply:
yes, keep it